modules/core/04-channel/v2/keeper/msg_server.go

```diff
@@ -17,7 +17,50 @@ import (
 
 var _ channeltypesv2.MsgServer = &Keeper{}
 
-// SendPacket implements the PacketMsgServer SendPacket method.
+// CreateChannel defines a rpc handler method for MsgCreateChannel.
```

Comment:
Moved CreateChannel and RegisterCounterparty to top of file so things read logically wrt exec flow

Reply:
say no more!